content/sql/concepts/operators/terms/exists/exists.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'EXISTS'
+Description: 'EXISTS tests a subquery and returns TRUE if at least one record satisfies it.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Operators'
+  - 'Database'
+  - 'Queries'
+  - 'SQLite'
+  - 'MySQL'
+  - 'PostgreSQL'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+  - 'paths/design-databases-with-postgresql'
+---
+
+`EXISTS` tests a subquery and returns `TRUE` if at least one record satisfies it. Otherwise, `FALSE` is returned.
+
+## Syntax
+
+```pseudo
+SELECT column1, column2, ...
+FROM table_one
+WHERE EXISTS (SELECT column1 FROM table_two WHERE condition);
+```
+
+## Example
+
+Return all rows from table `users` where `user_id` appears in table `locked_out`:
```

Comment:
```suggestion
In the example below, the statement returns `TRUE` if any and all rows (or a given user) in the `users` table that also exists in the `locked_out` table, per a match in `user_id`. Otherwise, if no such row exists, the statement returns `FALSE`.
```